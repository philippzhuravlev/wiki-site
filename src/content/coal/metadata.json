{
    "regions": [
      {
        "id": "angbria",
        "equivalents": {
          "golden": "valloraich",
          "steel": "albrion"
        }
      },
      {
        "id": "allemannland",
        "equivalents": {
          "golden": "theutoland",
          "steel": "carowiga"
        }
      }
    ]
  } 