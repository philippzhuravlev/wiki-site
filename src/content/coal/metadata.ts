interface RegionMetadata {
    id: string;
    equivalents: Record<string, string>;
    pageContentDir: string;
  }
  
  export const metadata: RegionMetadata[] = [
    {
      id: "dunemere",
      equivalents: {
        "golden-period": "valloraich",
        "steel": "albrion"
      },
      pageContentDir: "pages"
    },
    {
      id: "grimstone",
      equivalents: {
        "golden-period": "angbria",
        "steel": "theutoland"
      },
      pageContentDir: "pages"
    },
    // Add more regions...
  ];