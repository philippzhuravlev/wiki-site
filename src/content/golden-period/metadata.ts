interface RegionMetadata {
  id: string;
  equivalents: Record<string, string>;
  pageContentDir: string;
}

export const metadata: RegionMetadata[] = [
  {
    id: "valloraich",
    equivalents: {
      "steel": "albrion",
      "coal": "angbria"
    },
    pageContentDir: "pages"
  },
  // Add more regions...
]; 