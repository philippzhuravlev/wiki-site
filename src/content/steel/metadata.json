{
  "regions": [
    {
      "id": "albrion",
      "equivalents": {
        "golden": "valloraich",
        "coal": "angbria"
      }
    },
    {
      "id": "carowiga",
      "equivalents": {
        "golden": "theutoland",
        "coal": "allemannland"
      }
    }
  ]
} 