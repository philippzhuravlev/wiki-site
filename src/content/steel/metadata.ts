interface RegionMetadata {
    id: string;
    equivalents: Record<string, string>;
    pageContentDir: string;
  }
  
  export const metadata: RegionMetadata[] = [
    {
      id: "albrion",
      equivalents: {
        "golden-period": "valloraich",
        "coal": "dunemere"
      },
      pageContentDir: "pages"
    },
    {
      id: "theutoland",
      equivalents: {
        "golden-period": "angbria",
        "coal": "grimstone"
      },
      pageContentDir: "pages"
    },
    // Add more regions...
  ];