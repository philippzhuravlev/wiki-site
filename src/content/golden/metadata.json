{
    "regions": [
      {
        "id": "valloraich",
        "equivalents": {
          "steel": "albrion",
          "coal": "angbria"
        }
      },
      {
        "id": "theutoland",
        "equivalents": {
          "steel": "carowiga",
          "coal": "allemannland"
        }
      }
    ]
  } 